core/frontend/src/LocalhostIpcApp.ts

```diff
@@ -53,6 +53,6 @@ export class LocalhostIpcApp {
     IpcWebSocket.transport = new LocalTransport(opts?.localhostIpcApp?.socketPort ?? 3002);
     const ipc = new IpcWebSocketFrontend();
     await IpcApp.startup(ipc, opts);
-    await WebViewerApp.startup(opts); // this also attempts to initialize IModelApp, that's ok.
+    // await WebViewerApp.startup(opts); // this also attempts to initialize IModelApp, that's ok.
```

Comment:
Lets delete this line along with WebViewerApp

Reply:
Done